draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
Inspired by this, I learned a new appreciation (read: loathing) for ruby in the course of adding DOI support to kramdown.  It ended up consequential to solving a bunch of other problems.

See https://github.com/cabo/kramdown-rfc2629/pull/35